openmdao/docs/_utils/docutil.py

```diff
@@ -191,10 +191,11 @@ def remove_initial_empty_lines_from_source(source):
 # pylint: disable=C0103
 
 
-def get_source_code_of_class_or_method(class_or_method_path):
-    '''The function to be called a the custom Sphinx directive code
+def get_source_code_of_class_or_method(class_or_method_path, remove_docstring=True):
+    """
+    The function to be called a the custom Sphinx directive code
```

Comment:
This comment is poorly worded.  Oh wait, you inherited that comment.

Reply:
Fixed the comment. We really should have policed the style and pep stuff while these where being written.